scripts/create-fuels-template-integration.sh

```diff
@@ -1,7 +1,64 @@
 #!/bin/bash
 
-(cd templates/nextjs && pnpm run fuels:dev) &
-cp templates/nextjs/.env.example templates/nextjs/.env.local
-(cd templates/nextjs && pnpm run dev) &
+# Debugging
+# PUBLISHED_NPM_VERSION="next"
+
+# Versions
+FUEL_CORE_VERSION="0.28.0"
```

Comment:
Will these have to be manually updated every upgrade? We should be able to retrieve it from https://github.com/FuelLabs/fuels-ts/blob/master/internal/fuel-core/VERSION#L0-L1 no?